src/EnergyPlus/Pipes.hh

```diff
@@ -72,24 +73,24 @@ namespace Pipes {
 
         // Members
         std::string Name;
-        int TypeOf;        // type of pipe
-        int InletNodeNum;  // Node number on the inlet side of the plant
-        int OutletNodeNum; // Node number on the inlet side of the plant
-        int LoopNum;       // Index of plant loop where this pipe resides
-        int LoopSide;      // Index of plant loop side where this pipe resides
-        int BranchIndex;   // Index of plant Branch index where this pipe resides
-        int CompIndex;     // Index of plant Comp index where this pipe resides
+        DataPlant::PlantEquipmentType TypeOf; // type of pipe
+        int InletNodeNum;                     // Node number on the inlet side of the plant
+        int OutletNodeNum;                    // Node number on the inlet side of the plant
+        int LoopNum;                          // Index of plant loop where this pipe resides
```

Comment:
These four should always be collected into the `PlantLocation` struct. Make that your next PR. Not even joking.

Reply:
Will do!

Reply:
[this](https://github.com/NREL/EnergyPlus/wiki/Parameters-and-Return-Values)